tests/components/iqvia/conftest.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 
-from homeassistant.components.iqvia.const import CONF_ZIP_CODE, DOMAIN
+from homeassistant.components.iqvia import CONF_ZIP_CODE, DOMAIN
```

Comment:
These are not explicitly re-exported there.

Reply:
The same concept was approved here: https://github.com/home-assistant/core/pull/64608

Reply:
In that case, it should not have been approved. We are working towards getting mypy to enforce at this point.

Either it should be re-exported properly, or imported from the original source.

Reply:
Background information: <https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport>

Reply:
These are tests for the iqvia integration, which tests internals of the iqvia.
It should actually be using internals of the integration; As this is about defining the public API; there is no public use of the CONF_ZIP_CODE in this case.

Reply:
I don't have a particular horse in the race, so I'm open. To confirm, you're recommending:

1. Cancel this PR
2. Revert #64608 

?

Reply:
Yes